org.eclipse.wb.core/src/org/eclipse/wb/internal/core/editor/structure/property/ComponentsPropertiesPage.java

```diff
@@ -527,16 +525,14 @@ private void update_showEventsAction() throws Exception {
    * Shows {@link Property}'s of current objects.
    */
   private void refreshProperties() {
-    ExecutionUtils.runLog(new RunnableEx() {
-      public void run() throws Exception {
-        if (m_showEvents) {
-          showEvents();
-        } else {
-          showProperties();
-        }
-        // update actions
-        updateActions();
+    ExecutionUtils.runLog(() -> {

```

Comment:
Please send other separate PR for the annymous class to lambda conversion. You can do this via another cleanup, tab Code Style.